fvm/fvm_bench_test.go

```diff
@@ -348,21 +363,26 @@ func BenchmarkRuntimeTransaction(b *testing.B) {
 			}
 			`)
 
-		serviceAccount.AddArrayToStorage(b, blockExecutor, []string{longString, longString, longString, longString, longString})
```

Comment:
q: `serviceAccount` does not need this even when we are using service account to do the work?

Reply:
looks like this is only used for setting up copy / borrowing arrary bench tests.  those now run under regular accounts